lstchain/reco/r0_to_dl1.py

```diff
@@ -356,23 +356,22 @@ def r0_to_dl1(
         # Forcing filters for the dl1 dataset that are currently read from the pre-existing files
         # This should be fixed in ctapipe and then corrected here
         writer._h5file.filters = filters
-        print("USING FILTERS: ", writer._h5file.filters)
+        logger.info("USING FILTERS: ", writer._h5file.filters)
 
-        first_valid_ucts = None
-        first_valid_ucts_tib = None
+        first_valid_ucts = np.nan
+        first_valid_ucts_tib = np.nan
         previous_ucts_time_unix = []
         previous_ucts_trigger_type = []
 
         for i, event in enumerate(chain([first_event],  event_iter)):
 
             if i % 100 == 0:
-                print(i)
+                logger.info(i)
```

Comment:
You could just use a real progress bar like `tqdm`?

Reply:
Will try. There are though some other messages from muon rings mixed with the events evolution:

```
...
700
Impact parameter=3.978 m, ring_width=0.073 deg, ring radius=1.075 deg, ring completeness=0.700
800
...
```
The message does not come from this script itself but somewhere else.

Would that be a problem with the progress bar?

Reply:
It will then span over muttiple lines, creating a newline each time a non-progressbar message is printed. But it will still work.

Reply:
OK. To be implemented in a new PR

Reply:
we should get rid of these muon messages or improve them...